Libraries/LibJS/Runtime/ArrayConstructor.cpp

```diff
@@ -41,6 +41,9 @@ ArrayConstructor::ArrayConstructor()
 {
     put("prototype", interpreter().global_object().array_prototype(), 0);
     put("length", Value(1), Attribute::Configurable);
+
+    u8 attr = Attribute::Writable | Attribute::Configurable;
```

Comment:
Just curious, why is this writable?
and for that matter, why is `length` configurable?

MDN doesn't specify, but v8 says what you're saying here.
I don't get why this alone would be writable though

Reply:
I tried to think of a good answer but I couldn't, and the spec doesn't tell either. So probably: because that's what other engines have been doing before us? :/

Reply:
Fantastic :)